frontend/bin/www

```diff
@@ -24,8 +24,7 @@ var server = http.createServer(app);
 /**
  * Listen on provided port, on all network interfaces.
  */
-
-server.listen(port);
+server.listen(port, '0.0.0.0');
```

Comment:
We're probably not going to add this, since it makes modeldb available to all local network users.

We'll instead add a comment explaining how one might do it.